communication/slack-config/channels.yaml

```diff
@@ -291,6 +291,7 @@ channels:
   - name: kudo
   - name: kurl
   - name: kustomize
+  - name: kustomize-training-cohort # archive once cohort finishes
```

Comment:
Is there some configuration we need to add here to make this a private channel? 

cc @mrbobbytables

Reply:
Would love @mrbobbytables feedback.

I wrote this based on previous PRs I saw. I know that we can also specify members?